wire-ios-sync-engine/Source/Data Model/Conversation+Join.swift

```diff
@@ -175,8 +186,14 @@ struct ConversationJoinRequestFactory {
 
         var url = URLComponents()
         url.path = joinConversationsPath
-        url.queryItems = [URLQueryItem(name: URLQueryItem.Key.conversationKey, value: key),
-                          URLQueryItem(name: URLQueryItem.Key.conversationCode, value: code)]
+
+        let queryItems = [
+            URLQueryItem(name: URLQueryItem.Key.conversationKey, value: key),
+            URLQueryItem(name: URLQueryItem.Key.conversationCode, value: code)
+        ]
+
+        url.queryItems = queryItems
```

Comment:
Can it be formatted without extra `let`?
```suggestion
        url.queryItems = [
            URLQueryItem(name: URLQueryItem.Key.conversationKey, value: key),
            URLQueryItem(name: URLQueryItem.Key.conversationCode, value: code)
        ]
```